tests/test_robots.py

```diff
@@ -368,6 +368,20 @@ def hook(response):
                 'http://example.com/robots.txt', after_response_hook=hook)
             self.assertTrue(state["called"])
 
+    def test_after_response_hook_on_error(self):
+        '''Calls after_response_hook when error occurs during fetch'''
+        state = {"called": False}
+        expected_url = 'http://localhost:8080/robots.txt'
```

Comment:
Will this test fails if running on a box which happens to have a server running on port 8080? Maybe make it a more obsure port.

Reply:
`reppy` tests run in a Vagrant image, so the port should always be off inside the vagrant box and in Travis. FWIW, this pattern was copied from an existing test:

https://github.com/seomoz/reppy/blob/751e7324e443111b32f40975751ba0c8ccc3cd58/tests/test_robots.py#L220-L223